VAMobile/src/screens/BenefitsScreen/ClaimsScreen/ClaimDetailsScreen/ClaimStatus/ClaimFileUpload/TakePhotos/TakePhotos.tsx

```diff
@@ -51,7 +50,7 @@ function TakePhotos({ navigation, route }: TakePhotosProps) {
   }
 
   const collapsibleContent = (
```

Comment:
Could be further simplified adding the TextView stuff to `description` on the Alert and passing the LinkWithAnalytics directly as children.